src/types.jl

```diff
@@ -65,6 +65,38 @@ bioclim_key(k::Integer) = keys(bioclim_lookup)[k]
 bioclim_int(k::Integer) = k
 bioclim_int(k::Symbol) = bioclim_lookup[bioclim_key(k)]
 
+"""
+    BioClimPlus <: RasterDataSet
+
+Extended BioClim datasets, available from CHELSA. 
+More information on the CHELSA website: https://chelsa-climate.org/exchelsa-extended-bioclim/
+
+Some of these are available as average annual maximum, minimum, mean, and range. 
+Others have a single value, more like the regular BioClim variables.
+
+They do not usually use `month` or `date` keywords, but may use
+`date` in past/future scenarios. 
+
+Currently implemented for CHELSA as `CHELSA{BioClim}` and `CHELSA{Future{BioClim, args..}}`,
+specifying layer names as `Symbol`s.
+
+See the [`getraster`](@ref) docs for implementation details.
+"""
+struct BioClimPlus <: RasterDataSet end
+
+layerkeys(T::Type{BioClimPlus}) = vcat(collect(layerkeys(BioClim)), bioclimplus_layerkeys)
+
+const _bioclimplus_monthly = (:hurs, :clt, :sfcWind, :vpd, :rsds, :pet, :cmi, :swb)
```

Comment:
Good to make constants UPPER_CASE so we know where they come from when you use them

Reply:
See the point. `bioclim_lookup` is lowercase, so I tried to keep that consistent.

Reply:
Should be uppercase too if its a constant ;)